services/web-server/package.json

```diff
@@ -43,13 +21,11 @@
     "dataloader": "^1.4.0",
     "debug": "^3.1.0",
     "deepmerge": "^2.0.1",
-    "ejs": "^2.6.1",
```

Comment:
Is there a reason for removing this package? https://github.com/taskcluster/taskcluster-web-server/blob/c5442223d2e6994a8bfc928afa13ae7a48ff4bc3/src/servers/createApp.js#L12 uses it.

Reply:
Weird - our metatests ruled this dependency to be unused. Good catch here, thank you!